### DISCUSSION TITLE
No power and Cadence with S22i

### DISCUSSION THREAD

**Discussion by stephanelamey** [cagnulein/qdomyos-zwift#1535] [General]
Good morning; I have an S22i bike, I'm stuck at step 5. Qdomyos dashboard values remain zero when I run a manual workout on IFIT. I manage to have an inclination value appear provided that it is an integer value: if 3% on IFIT, ok 3% on Qdomyos. If 3.5% on IFIT then 0% on Qdomyos. All power and cadence values remain at zero. I have a garmin HRM, the values are well displayed by Qdomyos. With each change of parameter, I restarted the application, but nothing helped. Thank you for your help. best regards

**Comment by cagnulein** [comment]
hi @stephanelamey could you please provide me a debug log https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

thanks

**Reply by stephanelamey** [reply]
Hi Roberto,
See file attached
Thanks
Regards
Stephane LAMEY
[debug-lun__juin_26_04_20_36_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/11862717/debug-lun__juin_26_04_20_36_2023.log)

**Comment by stephanelamey** [comment]
Hi Roberto,
See file attached
Thanks
Regards
Stephane LAMEY
[debug-lun__juin_26_04_20_36_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/11862713/debug-lun__juin_26_04_20_36_2023.log)

**Comment by cagnulein** [comment]
@stephanelamey the issue is that you used the ip in the treadmill setting instead you have to use it in the proform bike setting, tdf companion ip. remove it from the treadmill one and press ok and put in the bike one and press ok and restart qz. Let me know

**Reply by stephanelamey** [reply]
Thank you Roberto
I'll look at it at noon, I'll let you know.
I have a question, in addition to the Norditrack S22i spinning bike (for 2 years), I have a Home Trainer Wahoo Knick Bike V6 (for 6 months). Can I use QZDomyos with the Wahoo? if so, how do I retrieve the Wahoo's IP address and enter it into the QZDomyos application?
I also have a Wahoo Headwind fan, can I control it with QZDomyos and how please.
Thanks Roberto.
Regards

**Reply by cagnulein** [reply]
@stephanelamey yes Kicr bike is already supported (using bluetooth, so you just need to create different profile in QZ to set the manual device in the advanced setting to that bike). And yes also the headwind is supported in the beta check #1281 

Let me know!

**Reply by stephanelamey** [reply]
Hi Roberto,
Regarding my S22i bike, the inclination display problem persists. QZdomyos only shows me the integer values 1,2,3... and not the intermediate values 1.5, 2.5, 3.5 .... it gives me the value 0.
I have the power and cadence value, but the speed remains at 0.
I tested by disabling and enabling speed based on power, always 0.
I attach the 2 debug files without and with speed based on power.
THANKS
Regards
Stephane LAMEY
[debug-lun__juin_26_11_50_27_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/11867033/debug-lun__juin_26_11_50_27_2023.log)
[debug-lun__juin_26_11_52_48_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/11867035/debug-lun__juin_26_11_52_48_2023.log)

**Comment by cagnulein** [comment]
hi @stephanelamey i guess i already fixed those in the beta. 
send me an email to roberto.viola83 at gmail.com indicating this ticket and if you are on android or iOS and i will send you it! Thanks
